sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -145,26 +145,28 @@ plot as shown below:
 Jupyter Notebooks
 =================
 
-At this point we should mention Jupyter (formerly IPython) notebooks.
-
-Jupyter notebooks are a fantastic tool for exploratory data analysis, and they
-are widely used across the "PyData" community. Bokeh integrates seamlessly with
-Jupyter notebooks. To view the above examples in a notebook, you would only change
-|output_file| to a call to |output_notebook| instead.
+Jupyter notebooks are a fantastic tool for exploratory data analysis, widely 
+used across the "PyData" community. They were previously known as IPython 
+notebooks. Bokeh integrates seamlessly with Jupyter notebooks. To view the 
+above examples in a notebook, you would only change |output_file| to a call 
+to |output_notebook| instead.
 
-A large number of static examples may be viewed directly online at the
+Many static examples may be viewed directly online at the
 `Bokeh NBViewer Gallery`_.
 
 The `Bokeh GitHub repository`_ also has a number of example notebooks in the
-``examples/plotting/notebook/`` directory. After cloning the repository,
-navigate there and run::
+``examples/howto`` directory. After cloning the repository, navigate there and run::
 
     ipython notebook
```

Comment:
suggest changing this to `code-block` format like everywhere else as long as we are editing here